test_projects/django/testproj/someapp/tasks.py

```diff
@@ -1,8 +1,14 @@
 import string
 import random
 
+from django.utils import six
+
 from jobtastic import JobtasticTask
 
+if six.PY3:
+    per_character_factor = 4  # py3 unicode characters take less memory
```

Comment:
Thank you!
I still have a nagging feeling the leaky global bloating could be done better. A leaky task with a bloat factor of 5 should be leaking about 5 MB of memory, but it leaks at least twice that.

But the leak sizes now roughly match between py2 and py3, so tests are working. So... meh.